picamera2/picamera2.py

```diff
@@ -270,7 +270,7 @@ def _reset_flags(self) -> None:
         self.frames = 0
         self._job_list = []
         self.options = {}
-        self._encoder = None
+        self._encoder = set()
```

Comment:
Good point, will change that.  Just wondering with the encoders property, should that let you assign a set of encoders, that overwrites the previous set of encoders now?

Reply:
Not sure! Currently you have to use `start_encoder`, but that also sets up a whole load of stuff like the size etc. What would you do with an encoder that hadn't been set up "properly"? Perhaps it should only be for checking what encoders it thinks are running?

Reply:
I've just made it so that you can assign a single encoder, or a set of additional encoders via the property, as a couple of examples used that method.